src/Types.hs

```diff
@@ -27,38 +27,63 @@ import qualified Data.Csv as CSV
 
 -- | Globally Unique ID (for reference in the database).  (FIXME: should we have different id types
 -- for different object types?)
-type GUID = Integer
+newtype GUID a = GUID Integer
```

Comment:
@fhartwig: agreed.

@Mikolaj: right.

So...  `UID`?  I'll make it `UID`?  Sounds like user id.  `Handle`?  No!  Any ideas?